lib/manager/regex/index.ts

```diff
@@ -1,55 +1,161 @@
 import { logger } from '../../logger';
 import { regEx } from '../../util/regex';
 import * as template from '../../util/template';
-import { CustomExtractConfig, PackageFile, Result } from '../common';
+import {
+  CustomExtractConfig,
+  PackageDependency,
+  PackageFile,
+  Result,
+} from '../common';
 
 export const defaultConfig = {
   pinDigests: false,
 };
 
-export function extractPackageFile(
-  content: string,
-  packageFile: string,
-  config: CustomExtractConfig
-): Result<PackageFile | null> {
-  const regexMatch = regEx(config.matchStrings[0], 'g');
-  const deps = [];
+const fields = [
```

Comment:
```suggestion
const validMatchFields = [
```

Use a more descriptive name for easier maintenance.